server/host_controls.go

```diff
@@ -97,6 +97,36 @@ func (p *Plugin) muteSession(requesterID, channelID, sessionID string) error {
 	return nil
 }
 
+func (p *Plugin) muteOthers(requesterID, channelID string) error {
+	state, err := p.getCallState(channelID, false)
+	if err != nil {
+		return err
+	}
+
+	if state == nil {
+		return ErrNoCallOngoing
+	}
+
+	if requesterID != state.Call.GetHostID() {
+		if isAdmin := p.API.HasPermissionTo(requesterID, model.PermissionManageSystem); !isAdmin {
+			return ErrNoPermissions
+		}
+	}
+
+	// Unmute anyone muted (who is not the host/requester).
+	// If there are no unmuted sessions, return without doing anything.
+	for id, s := range state.sessions {
+		if s.Unmuted && s.UserID != requesterID {
+			p.publishWebSocketEvent(wsEventHostMute, map[string]interface{}{
+				"channel_id": channelID,
+				"session_id": id,
+			}, &model.WebsocketBroadcast{UserId: s.UserID, ReliableClusterSend: true})
+		}
+	}
```

Comment:
Could we add a TODO so I remember to update this when we finally merge https://github.com/mattermost/mattermost-plugin-calls/pull/609?

Reply:
@streamer45 Hm, I'm not sure how that would that affect this?

Reply:
It will because we are extending `publishWebSocketEvent` to support a slice of user ids to loop through which is essentially what we are doing here.

Reply:
Ahhhhh, yes. Thanks.